pages/pokemon/[name].jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { fetchPokemonDetails } from './../../apis';
+
+export async function getServerSideProps(context) {
+    const response = await fetchPokemonDetails(context.query.name);
+    return {
```

Comment:
Prettier doesn't seem to be configured - spacing, etc is not consistent across file

Reply:
Done as stated